### ISSUE TITLE
loss和val_loss不更新

### ISSUE THREAD

**Issue by Jiangdan135** [tslgithub/image_class#21]
你好，感谢你提供的代码，我在跑VGG16,VGG19,AlexNet时都出现了结果不更新的情况，在ResNet50上结果正常，请问我该如何解决呢？
Epoch 1/10
12992/12992 [==============================] - 108s 8ms/step - loss: 1.7006 - acc: 0.8902 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00001: val_loss improved from inf to 1.67843, saving model to ./checkpoints/VGG19/VGG19.h5
Epoch 2/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00002: val_loss did not improve from 1.67843
Epoch 3/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00003: val_loss did not improve from 1.67843
Epoch 4/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00004: val_loss did not improve from 1.67843
Epoch 5/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00005: val_loss did not improve from 1.67843
Epoch 6/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00006: val_loss did not improve from 1.67843

Epoch 00006: ReduceLROnPlateau reducing learning rate to 0.0004999999888241291.
Epoch 7/10
12992/12992 [==============================] - 98s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00007: val_loss did not improve from 1.67843
Epoch 8/10
12992/12992 [==============================] - 99s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00008: val_loss did not improve from 1.67843
Epoch 9/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00009: val_loss did not improve from 1.67843
Epoch 10/10
12992/12992 [==============================] - 100s 8ms/step - loss: 1.7046 - acc: 0.8942 - val_loss: 1.6784 - val_acc: 0.8959

Epoch 00010: val_loss did not improve from 1.67843

**Comment by tslgithub** [comment]
调参数，学习率调小点

**Comment by Jiangdan135** [comment]
谢谢，请问运行preict.py后出现wrong label是什么情况呢，是图片.jpg的名字也必须是类名字吗？

**Comment by tslgithub** [comment]
图片名字可以随意，但是图片所在的”文件夹“的名字必须是类别的名字

**Comment by tslgithub** [comment]
wrong  label说明是预测错啦，比如文件夹下面的全部都是1类，但是预测时有些图片被预测成了2类，wrong  label是正常情况的

**Comment by Jiangdan135** [comment]
谢谢，但我还是不懂为什么学习率大反而不更新呢，学习率大变化幅度不是应该更大吗？

**Comment by Jiangdan135** [comment]
谢谢，但我还是不懂为什么学习率大反而不更新呢，学习率大变化幅度不是应该更大吗？
